tensai-api/src/main/java/dev/vmsa/tensai/utils/Tuple.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of tensai, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) 2022 VMSA
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package dev.vmsa.tensai.utils;
+
+/**
+ * <p>Tuples are used to store 2 values of different or same types in a single object. Mainly used as
+ * method return type.</p>
+ *
+ * @param <A> First tuple type.
+ * @param <B> Second tuple type.
+ */
+public class Tuple<A, B> {
```

Comment:
Already have ig

Reply:
This class is for ``tensai-api``, which is platform-independent.

I actually redesigned ``EventEmitter`` to not use ``Tuple``, but I'm waiting for your response in https://github.com/vmsa-dev/tensai/pull/4#discussion_r1059652486.

Reply:
Minecraft has commons lang as dep, can use from that

Reply:
There's no ``Tuple`` for Spigot but we also don't need that anymore.